tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -16,22 +16,23 @@
 import itertools
 from collections import namedtuple
 
+DATA_FILES_PATH = '../framework/data_files'
 # define certificates configuration entry
 Certificate = namedtuple("Certificate", ['cafile', 'certfile', 'keyfile'])
 # define the certificate parameters for signature algorithms
 CERTIFICATES = {
-    'ecdsa_secp256r1_sha256': Certificate('data_files/test-ca2.crt',
-                                          'data_files/ecdsa_secp256r1.crt',
-                                          'data_files/ecdsa_secp256r1.key'),
-    'ecdsa_secp384r1_sha384': Certificate('data_files/test-ca2.crt',
-                                          'data_files/ecdsa_secp384r1.crt',
-                                          'data_files/ecdsa_secp384r1.key'),
-    'ecdsa_secp521r1_sha512': Certificate('data_files/test-ca2.crt',
-                                          'data_files/ecdsa_secp521r1.crt',
-                                          'data_files/ecdsa_secp521r1.key'),
-    'rsa_pss_rsae_sha256': Certificate('data_files/test-ca_cat12.crt',
-                                       'data_files/server2-sha256.crt', 'data_files/server2.key'
-                                       )
+    'ecdsa_secp256r1_sha256': Certificate(DATA_FILES_PATH + '/test-ca2.crt',
```

Comment:
Rather
```suggestion
    'ecdsa_secp256r1_sha256': Certificate('$DATA_FILES_PATH/test-ca2.crt',
```
I'd say with DATA_FILES_PATH defined in ssl-opt.sh and then update tls13-compat.sh accordingly.